### ISSUE TITLE
There are many databasex.log when testing the latest MySQL.

### ISSUE THREAD

**Issue by qaqcatz** [sqlancer/sqlancer#569]
Hello, I'm testing the latest MySQL(MYSQL_VERSION=8.0.30-1.el8):

```
sudo docker run -itd --name mysql -p 13306:3306 -e MYSQL_ROOT_PASSWORD=123456 mysql
```

 with sqlancer [v2.0.0](https://github.com/sqlancer/sqlancer/tree/v2.0.0):

```
--num-queries 500 --num-threads 16 --timeout-seconds 100 --port 13306 --username root --password 123456 mysql --oracle PQS
```

There are 66 `databasex.log` in all 80 `databasex-cur.log` after the test.

 All 66 `databasex.log` are `java.lang.AssertionError`. They can be divided into 3 categories according to `Caused by`:
1. Caused by: java.sql.SQLException: Unknown system variable 'myisam_repair_threads'
```
--java.lang.AssertionError: SET SESSION myisam_repair_threads = 7155401647472894619;
--	at sqlancer.common.query.SQLQueryAdapter.checkException(SQLQueryAdapter.java:100)
--	at sqlancer.common.query.SQLQueryAdapter.execute(SQLQueryAdapter.java:93)
--	at sqlancer.Main$QueryManager.execute(Main.java:239)
--	at sqlancer.GlobalState.executeStatement(GlobalState.java:108)
--	at sqlancer.StatementExecutor.executeStatements(StatementExecutor.java:70)
--	at sqlancer.mysql.MySQLProvider.generateDatabase(MySQLProvider.java:148)
--	at sqlancer.mysql.MySQLProvider.generateDatabase(MySQLProvider.java:1)
--	at sqlancer.ProviderAdapter.generateAndTestDatabase(ProviderAdapter.java:40)
--	at sqlancer.Main$DBMSExecutor.run(Main.java:327)
--	at sqlancer.Main$2.run(Main.java:511)
--	at sqlancer.Main$2.runThread(Main.java:489)
--	at sqlancer.Main$2.run(Main.java:479)
--	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
--	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
--	at java.base/java.lang.Thread.run(Thread.java:833)
--Caused by: java.sql.SQLException: Unknown system variable 'myisam_repair_threads'
--	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
--	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
--	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:768)
--	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:653)
--	at sqlancer.common.query.SQLQueryAdapter.execute(SQLQueryAdapter.java:87)
--	... 13 more
```
2. Caused by: java.sql.SQLSyntaxErrorException: The valid range for key_block_size is [0,65535]
```
--java.lang.AssertionError: CREATE TABLE t0(c0 MEDIUMINT(168)   STORAGE DISK NULL COMMENT 'asdf'  UNIQUE) MIN_ROWS = 9202107253323221330, KEY_BLOCK_SIZE = 5516652755378908524;
--	at sqlancer.common.query.SQLQueryAdapter.checkException(SQLQueryAdapter.java:100)
--	at sqlancer.common.query.SQLQueryAdapter.execute(SQLQueryAdapter.java:93)
--	at sqlancer.Main$QueryManager.execute(Main.java:239)
--	at sqlancer.GlobalState.executeStatement(GlobalState.java:108)
--	at sqlancer.mysql.MySQLProvider.generateDatabase(MySQLProvider.java:139)
--	at sqlancer.mysql.MySQLProvider.generateDatabase(MySQLProvider.java:1)
--	at sqlancer.ProviderAdapter.generateAndTestDatabase(ProviderAdapter.java:40)
--	at sqlancer.Main$DBMSExecutor.run(Main.java:327)
--	at sqlancer.Main$2.run(Main.java:511)
--	at sqlancer.Main$2.runThread(Main.java:489)
--	at sqlancer.Main$2.run(Main.java:479)
--	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
--	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
--	at java.base/java.lang.Thread.run(Thread.java:833)
--Caused by: java.sql.SQLSyntaxErrorException: The valid range for key_block_size is [0,65535]. Error near '5516652755378908524' at line 1
--	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
--	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
--	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:768)
--	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:653)
--	at sqlancer.common.query.SQLQueryAdapter.execute(SQLQueryAdapter.java:87)
--	... 12 more
```
3. Caused by: java.sql.SQLSyntaxErrorException: The valid range for avg_row_length is [0,4294967295]
```
--java.lang.AssertionError: CREATE TABLE t0(c0 MEDIUMTEXT  COLUMN_FORMAT DYNAMIC STORAGE MEMORY, c1 TINYTEXT  COMMENT 'asdf'   STORAGE DISK COLUMN_FORMAT DYNAMIC) AVG_ROW_LENGTH = 4873203941820172821, STATS_AUTO_RECALC = 0;
--	at sqlancer.common.query.SQLQueryAdapter.checkException(SQLQueryAdapter.java:100)
--	at sqlancer.common.query.SQLQueryAdapter.execute(SQLQueryAdapter.java:93)
--	at sqlancer.Main$QueryManager.execute(Main.java:239)
--	at sqlancer.GlobalState.executeStatement(GlobalState.java:108)
--	at sqlancer.mysql.MySQLProvider.generateDatabase(MySQLProvider.java:139)
--	at sqlancer.mysql.MySQLProvider.generateDatabase(MySQLProvider.java:1)
--	at sqlancer.ProviderAdapter.generateAndTestDatabase(ProviderAdapter.java:40)
--	at sqlancer.Main$DBMSExecutor.run(Main.java:327)
--	at sqlancer.Main$2.run(Main.java:511)
--	at sqlancer.Main$2.runThread(Main.java:489)
--	at sqlancer.Main$2.run(Main.java:479)
--	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
--	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
--	at java.base/java.lang.Thread.run(Thread.java:833)
--Caused by: java.sql.SQLSyntaxErrorException: The valid range for avg_row_length is [0,4294967295]. Error near '4873203941820172821, STATS_AUTO_RECALC = 0' at line 1
--	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
--	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
--	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:768)
--	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:653)
--	at sqlancer.common.query.SQLQueryAdapter.execute(SQLQueryAdapter.java:87)
--	... 12 more
```

Is this normal? Or is there something wrong with my parameters?

**Comment by mrigger** [comment]
This is not expected. The reason for this is that SQLancer hasn't been upgraded to the latest MySQL version. If you want to install the currently supported MySQL version (8.0.20), you could execute the following commands:
https://github.com/sqlancer/sqlancer/blob/4c6dd45305eb3e546dca703020450387cb5cf7bb/.github/workflows/main.yml#L194-L201

Alternatively, you (or someone else) could try to upgrade SQLancer to the latest MySQL version. It seems that MySQL now longer accepts arbitrary integers for the options in the test cases you posted. To address the exception caused by the last test case you posted, you could generate an integer in the expected range here:
https://github.com/sqlancer/sqlancer/blob/4c6dd45305eb3e546dca703020450387cb5cf7bb/src/sqlancer/mysql/gen/MySQLTableGenerator.java#L181

I probably won't have enough time to fix this in the next weeks. Let me know if you want to work on it, and don't hesitate to reach out for further guidance if so.

**Comment by qaqcatz** [comment]
Thanks for your reply. I will try to fix it in mysql 8.0.20+. Hopefully I can submit a PR.

**Comment by malwaregarry** [comment]
Closing this as the current supported version in SQLancer is 8.0.36 https://github.com/sqlancer/sqlancer/pull/940
